src/storybook/stand-alone-documentaion/typography/text-styles/text-styles.jsx

```diff
@@ -13,137 +12,83 @@ export const TextStyles = () => {
       <VisualDescription
         className={CSS_BASE_CLASS}
         ariaLabel="H1"
-        title="Main heading (Poppins 30px bold)"
```

Comment:
Are we sure we don't need this "(Poppins 30px bold)"? I think this information might be relevant for designers - but if Meirav or Yael think otherwise, I'm good